lib/ruby_lsp/handler.rb

```diff
@@ -62,7 +62,14 @@ def handle(request)
             error = e
           end
 
-          @writer.write(id: request[:id], result: result) unless result == VOID
+          if error
+            @writer.write(
+              { id: request[:id],
+                error: { code: Constant::ErrorCodes::INTERNAL_ERROR, message: error.inspect, data: request.to_json }, }
+            )
```

Comment:
This style looks weird:
```suggestion
              {
                id: request[:id],
                error: { code: Constant::ErrorCodes::INTERNAL_ERROR, message: error.inspect, data: request.to_json },
              }
            )
```